data/json/items/tool/explosives.json

```diff
@@ -1615,5 +1615,61 @@
       "explosion": { "power": 69200, "shrapnel": { "casing_mass": 12000, "fragment_mass": 600 } }
     },
     "flags": [ "BOMB", "TRADER_AVOID", "DANGEROUS" ]
+  },
+  {
+    "id": "jug_bomb",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "jug bomb" },
+    "description": "A glass jug with gasoline spilled on the inside and a small pouch of oxidizer powder next to it, which upon mixing will create an incredible shock explosion.  Activating this will mix the two ingredients, after which it's best to throw it as it might go off in your hands!",
+    "weight": "1133 g",
+    "volume": "750 ml",
+    "price": 6000,
+    "price_postapoc": 1000,
+    "to_hit": -3,
+    "bashing": 4,
+    "material": [ "glass" ],
+    "symbol": "|",
+    "color": "white",
+    "explode_in_fire": false,
+    "explosion": { "power": 938 },
+    "use_action": {
+      "need_wielding": true,
+      "target": "jug_bomb_act",
+      "msg": "You mix the ingredients.",
+      "target_charges": 20,
+      "active": true,
+      "need_fire": 0,
+      "menu_text": "Mix the ingredients",
+      "type": "transform"
+    },
+    "flags": [ "BOMB" ]
+  },
+  {
+    "id": "jug_bomb_act",
+    "type": "TOOL",
+    "category": "weapons",
+    "name": { "str": "jug bomb (mixed)", "str_pl": "jug bombs (mixed)" },
+    "description": "A mixed jug bomb, it may explode any moment now.",
+    "weight": "1133 g",
+    "volume": "750 ml",
+    "price": 0,
+    "price_postapoc": 0,
+    "to_hit": -3,
+    "bashing": 4,
+    "material": [ "glass" ],
+    "symbol": "|",
+    "color": "white",
+    "turns_per_charge": 1,
+    "explode_in_fire": true,
+    "explosion": { "power": 150 },
```

Comment:
Why is the explosion power 150 in activated and 938 in unactivated? Also there should probably be at least some fragments since it's in a glass vessel, even if the vessel shatters on impact.